### ISSUE TITLE
[Request] Multiple Camera Setup

### ISSUE THREAD

**Issue by harvo900** [ju1ce/Mediapipe-VR-Fullbody-Tracking#9]
A Feature that allows you to add multiple cameras for better tracking at different angles would be very good. My room is quiet small and its hard to make due with just one camera as the FOV isn't high enough for me to move around. A feature to add multiple cameras and would help with this along-side tracking quality.
I'm not entirely sure how difficult this is to implement so sorry if this sounds like something that may be challenging to implement into this amazing software.

Much thanks for taking the time to read this,
- Harvey

**Comment by dallytaur** [comment]
I agree the biggest ting that d4vr has was not being able to link up more cams

**Comment by monsieurpooh** [comment]
I also strongly desire this feature which would take many applications to the next level. It is very important for use cases and games where you won't always be facing the camera, which tends to glitch out this particular model. Are there any plans to implement it? How much time would it take?

**Comment by monsieurpooh** [comment]
It occurs to me since this whole thing is based on https://developers.google.com/mediapipe/solutions/vision/pose_landmarker, this might be a question / feature request for Google
